nessie/src/main/java/org/apache/iceberg/nessie/NessieCatalog.java

```diff
@@ -183,7 +184,8 @@ private String validateWarehouseLocation(String name, Map<String, String> catalo
           catalogOptions);
       throw new IllegalStateException("Parameter 'warehouse' not set, Nessie can't store data.");
     }
-    return warehouseLocation;
+
+    return LocationUtil.stripTrailingSlash(warehouseLocation);
```

Comment:
Just making this inline with other catalogs. 
We can lookup the callers of `LocationUtil.stripTrailingSlash`